### PR TITLE
[Cognitive Services] Update auto-generated code for ComputerVision, Face, SpellCheck, TextAnalytics.

### PR DESCRIPTION
**Pull request by yangyuan**
[Cognitive Services] Update auto-generated code for ComputerVision, Face, SpellCheck, TextAnalytics.

For ComputerVision, Face, SpellCheck, TextAnalytics:
1. Update auto-generated code.
2. Update tests to fit changes (no test logic change).
3. Major version bump.

For SpellCheck:
Rename generate.ps1 to lower case, change "\" to "/"

For LUIS:
Replace generate.cmd to generate.ps1.

https://github.com/Azure/azure-rest-api-specs/pull/3351

## Description

---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [x] Please add REST spec PR link to the SDK PR
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [ ] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [x] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [x] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [x] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [x] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

**Comment by dsgouda** [comment]
@yangyuan Can you split this into 2 PRs
1. Just to modify the directory names
2. Newly generated code
This helps us with the bookkeeping and reviews

**Comment by dsgouda** [comment]
Closing this

**Comment by dsgouda** [comment]
Talked @yangyuan offline, this seems to be the best course of action due to the large number of SDK directories that need minor changes. Will post reviews soon

File: src/SDKs/CognitiveServices/dataPlane/Language/SpellCheck/BingSpellCheck/Microsoft.Azure.CognitiveServices.SpellCheck.csproj [hunk 1]
```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <PackageId>Microsoft.Azure.CognitiveServices.Language.SpellCheck</PackageId>
     <Description>This client library provides access to the Microsoft Cognitive Services SpellCheck API.</Description>
-    <Version>2.0.0</Version>
+    <Version>3.0.0</Version>
     <AssemblyName>Microsoft.Azure.CognitiveServices.Language.SpellCheck</AssemblyName>
     <PackageTags>Microsoft Cognitive Services;Cognitive Services;Cognitive Services SDK;SpellCheck API;REST HTTPclient;SpellCheckSDK;netcore451511</PackageTags>
     <PackageReleaseNotes>This is a public release of the Cognitive Services SpellCheck SDK. Included with this release is support for SpellCheck API.</PackageReleaseNotes>
```
Comment (dsgouda) [inline src/SDKs/CognitiveServices/dataPlane/Language/SpellCheck/BingSpellCheck/Microsoft.Azure.CognitiveServices.SpellCheck.csproj]: Please update `PackageReleaseNotes` here

**Review by dsgouda — CHANGES_REQUESTED** [review]
Looks great apart from a minor comment

File: src/SDKs/CognitiveServices/dataPlane/Language/SpellCheck/BingSpellCheck/Microsoft.Azure.CognitiveServices.SpellCheck.csproj (hunk 1 above)
Comment (yangyuan) [reply src/SDKs/CognitiveServices/dataPlane/Language/SpellCheck/BingSpellCheck/Microsoft.Azure.CognitiveServices.SpellCheck.csproj]: So the rule is: for those with "PackageReleaseNotes" tag, we should update it, for those without "PackageReleaseNotes" tag we do not have to add them. is that correct?

Comment (yangyuan) [reply src/SDKs/CognitiveServices/dataPlane/Language/SpellCheck/BingSpellCheck/Microsoft.Azure.CognitiveServices.SpellCheck.csproj]: @dsgouda cool, thanks for explanations.

**Review by dsgouda — APPROVED** [review]
LGTM will merge on CIs passing

**Pull request merged**